platforms/minigame/platforms/taobao-mini-game/wrapper/fs-utils.js

```diff
@@ -244,9 +244,8 @@ var fsUtils = {
                 onComplete && onComplete(new Error(`Failed to load subpackage ${name}: ${res.errorMessage}`));
             },
         });
-        task = task || {};
         // onProgress && task.onProgressUpdate(onProgress);
-        return task;
+        return {};
```

Comment:
why don't return task object

Reply:
The return value type of loadSubpackage is void